js/allDogs.js

```diff
@@ -0,0 +1,47 @@
+function Murphy() {
+    alert("Murphy, Dog, $123.45");
+  }
+
+function Poppy() {
+    alert("Poppy, Dog, $123.45");
+  }
+
+function Jack() {
+    alert("Jack, Dog, $123.45")
+}
+
+function Duffy() {
+    alert("Duffy, Dog, $123.45")
+}
+
+function Lucas() {
+    alert("Lucas, Dog, $123.45")
+}
+
+function Jake() {
+    alert("Jake, Dog, $123.45")
+}
+
+function Angus() {
```

Comment:
Good job, but think of how you could send data in from your click handler and reuse a function to do your work. Something like this..Also, you can keep all your JS to one file.

https://codepen.io/jimmynono/pen/RmbzQb?editors=1010